crapssim/strategy/tools.py

```diff
@@ -422,6 +422,37 @@ def __repr__(self) -> str:
         )
 
 
+class RemoveIfPointOff(RemoveIfTrue):
+    """Strategy that removes a bet if the table point is Off
+
+    This will match bets based on type, and number for Place and Hardway bets.
+    It will not consider bet amounts when matching."""
+
+    def __init__(self, bet: Bet):
+        if isinstance(bet, Place):
+            key = (
+                lambda b, p: isinstance(b, Place)
+                and b.number == self.bet.number
+                and p.table.point.status == "Off"
+            )
+        elif isinstance(bet, HardWay):
+            key = (
+                lambda b, p: isinstance(b, HardWay)
+                and b.number == self.bet.number
+                and p.table.point.status == "Off"
+            )
+        else:
+            key = (
+                lambda b, p: isinstance(b, type(self.bet))
+                and p.table.point.status == "Off"
+            )
+        super().__init__(key)
+        self.bet = bet
+
+    def __repr__(self) -> str:
+        return f"{self.__class__.__name__}(bet={self.bet})"
```

Comment:
Note that new `RemoveIfPointOff` strategy tool needs logic for any bets with numbers. This reflects a difference in identifying bets that could potentially be improved (i.e. with separate Place4, Place5, ..., etc bets) with more subclassing. Think about for future, but for now just make sure to incorporate any additions for new bets (buy, lay, coming from #38)